src/renderers/dom/client/syntheticEvents/SyntheticMouseEvent.js

```diff
@@ -63,6 +63,32 @@ var MouseEventInterface = {
       event.pageY :
       event.clientY + ViewportMetrics.currentScrollTop;
   },
+  offsetX: function(event) {
+    return 'offsetX' in event ?
+    event.offsetX :
+    'srcElement' in event ?
+      ((e) => {
+        const target = e.srcElement;
+        const style = target.currentStyle || window.getComputedStyle(target, null);
+        const borderLeftWidth = parseInt(style.borderLeftWidth, 10);
+        const rect = target.getBoundingClientRect();
+        return parseInt((e.clientX - borderLeftWidth - rect.left), 10);
```

Comment:
It is not needed on line 73. It is in the return statement because getBoundingClientRect.left returns a float. I used parseInt to match the MouseEvent offsetX, which returns an int.